### ISSUE TITLE
Bug! - Wordpress blogs are not added to the site correctly.

### ISSUE THREAD

**Issue by Snede** [daveaglick/discoverdotnet#299]
Hi!

My blog snede.net is not popping up when I publish new posts, and doesn't show up on the list of blogs either? What is the issue here and how can I fix it?

**Comment by SeanKilleen** [comment]
This intrigued me so I wanted to look into this and I figured it out! I'm not sure if it's all Wordpress blogs or just yours.

* I pulled the code for the web site according to the README file
* I deleted all of the blogs except yours & mine which I know happens to work correctly
* Followed the directions in readme and ran `build.cmd -target preview`
* Let the site build

In the site output, I see the following warning:

> Error getting feed for https://snede.net/feed: Response status code does not indicate success: 406 (Not Acceptable).

[Error 406 Not Acceptable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) means that the server can't produce content in the format that the request is asking for.

That struck me as odd, so I dug a little deeper. I opened up Fiddler and looked at the request.

The request is made, but sure enough, the body of the site does not respond as one might expect:

```
HTTP/1.1 406 Not Acceptable
Date: Mon, 01 Apr 2019 02:55:31 GMT
Server: Apache
Content-Length: 102
Content-Type: text/html; charset=iso-8859-1

<h1>Security incident detected</h1><p>Your request was blocked. Please try again later (or don't).</p>
```

I thought about this a little bit, and I realized that the Discover.NET build process isn't providing a user agent along with a request. I modified the request within Fiddler to provide a user agent and sure enough, it worked as expected.

So:

* **Symptom fix**: Update your security settings to allow a browser with no user agent
* **Root cause fix**: Modify the Discover .NET process to provide user agent along with its requests.

I'll submit a PR for the root cause fix shortly hopefully.
